### ISSUE TITLE
main app pipeline feedback from users

### ISSUE THREAD

**Issue by wassfila** [TheSoundOfAIOSR/project_common#33]
provided by @MKDYasserH , I copied it from the slack channel
[User-Experience-And-Error-Reports_Beat_Tödtli.docx](https://github.com/TheSoundOfAIOSR/project_common/files/7109643/User-Experience-And-Error-Reports_Beat_Todtli.docx)
[error_reporting_logs_archit.zip](https://github.com/TheSoundOfAIOSR/project_common/files/7109645/error_reporting_logs_archit.zip)
[logs_beat_toedtli.zip](https://github.com/TheSoundOfAIOSR/project_common/files/7109647/logs_beat_toedtli.zip)

**Comment by Leofltt** [comment]
Ticking last one (We propose there should be a help button/question mark [..].) as 'Won't do'. The button are doing exactly what is written, there is the need to know what a button does in a specific context, but that will be addressed with manual / video demo. There is no amount of description overlay that would actually be useful and not just cumbersome and we definitely don't have the capacity to dev a whole new feature like that and test it in a way that makes sense in time for eval. Maybe we can think about it later on

**Comment by wassfila** [comment]
right, prio is for crashes and dysfunctions, or user frustrations on not being able to run the main app.

**Comment by Leofltt** [comment]
Third issue (Sometimes the GUI would freeze. It would be good to know whether an expensive calculation is still running, such as the computation of wav files for pitches 46-88.) addressed by the latest merge to dev branch #36

**Comment by Leofltt** [comment]
Manual is in #40
